### PR TITLE
Julio valerio development

### PR DESCRIPTION
**Pull request by jvjvalerio**
Modify HelloWorldServlet to convert ArrayList to JSON.
Modify index.html to change About Me text.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
+//   public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
```
Comment (jonathantanmy) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Don't leave old code commented like this. If you want to see what the code originally looked like, you can look through the history in Git.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 2]
```diff
@@ -1,18 +1,43 @@
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+
+import org.eclipse.jetty.util.ajax.JSON;
+
 import java.io.IOException;
+import java.lang.reflect.Array;
```
Comment (jonathantanmy) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Try not to add unnecessary imports.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 1 above)
Comment (jvjvalerio) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Remove old commented code.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 2 above)
Comment (jvjvalerio) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Clean up imports.

**Review by jonathantanmy — APPROVED** [review]

**Pull request merged**